### ISSUE TITLE
Bug report.

### ISSUE THREAD

**Issue by Shelamkoff** [filp/whoops#750]
Hi.
After upgrading to version `2.15.0`, an exception is thrown when using the `Whoops\Handler\PrettyPageHandler` .
`[Exception: Error, File: vendor\filp\whoops\src\Whoops\Handler\PlainTextHandler.php, Line: 285, Message: Call to a member function getFrameFilters() on null, Code: 0]`
I believe this is related to the code below:
```php
// Whoops\Handler\PrettyPageHandler lines 289 - 292
$plainTextHandler = new PlainTextHandler();
$plainTextHandler->setException($this->getException());
$plainTextHandler->setInspector($this->getInspector());
$vars["preface"] = "<!--\n\n\n" .  $this->templateHelper->escape($plainTextHandler->generateResponse()) . "\n\n\n\n\n\n\n\n\n\n\n-->";
```
Calling `$plainTextHandler->setRun($this->getRun())` after initializing `$plainTextHandler` solves this problem

**Comment by follestad** [comment]
Same problem here.

**Comment by marcLeeroy** [comment]
same problem here

**Comment by AmraniCh** [comment]
Hey, thanks for opening this issue.
This error will be fixed after merging this PR  (#751) and releasing a new version.
